### ISSUE TITLE
Really strange number of line for problem MWoffliner

### ISSUE THREAD

**Issue by kelson42** [kiwix/metrics#58]
If I look to the stat of metrics.kiwix.org, it seems the project `openzim/mwoffliner` has around 20.000 lines of Typescript.
https://metrics.kiwix.org/app/kibana#/dashboard/Overview?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-5y,mode:quick,to:now))&_a=(description:'Overview%20Panel%20by%20Kiwix',filters:!(('$state':(store:appState),meta:(alias:'Empty%20Commits',disabled:!f,index:git,key:files,negate:!t,params:(query:'0',type:phrase),type:phrase,value:'0'),query:(match:(files:(query:'0',type:phrase)))),('$state':(store:appState),meta:(alias:Bots,disabled:!f,index:git,key:author_bot,negate:!t,params:(query:!t,type:phrase),type:phrase,value:!t),query:(match:(author_bot:(query:!t,type:phrase)))),('$state':(store:appState),meta:(alias:!n,controlledBy:'1577791253307',disabled:!f,index:'26d36150-0f7c-11ea-ae8c-d9e77f11fa16',key:origin,negate:!f,params:(query:'https:%2F%2Fgithub.com%2Fopenzim%2Fmwoffliner',type:phrase),type:phrase,value:'https:%2F%2Fgithub.com%2Fopenzim%2Fmwoffliner'),query:(match:(origin:(query:'https:%2F%2Fgithub.com%2Fopenzim%2Fmwoffliner',type:phrase))))),fullScreenMode:!f,options:(darkTheme:!f,useMargins:!t),panels:!((gridData:(h:3,i:'37',w:6,x:0,y:9),id:f62d9e20-184b-11ea-8930-836c4cc0aa6f,panelIndex:'37',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'40',w:6,x:6,y:0),id:bc6428d0-2bbf-11ea-b387-dfc81c1b31f0,panelIndex:'40',title:Filters,type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'41',w:6,x:6,y:3),id:'9740f5d0-2d4d-11ea-b387-dfc81c1b31f0',panelIndex:'41',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'43',w:6,x:6,y:9),id:'1b35a160-2d4e-11ea-b387-dfc81c1b31f0',panelIndex:'43',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'44',w:6,x:6,y:12),id:'95a63c20-2d4e-11ea-b387-dfc81c1b31f0',panelIndex:'44',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'46',w:6,x:6,y:6),id:'6e710420-2eff-11ea-b7fb-dfaec3b825cc',panelIndex:'46',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'50',w:6,x:0,y:3),id:'2ecb82b0-3bb5-11ea-95ab-5b7a00a159a1',panelIndex:'50',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'51',w:6,x:0,y:0),id:a69b2940-3bb4-11ea-95ab-5b7a00a159a1,panelIndex:'51',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'52',w:6,x:0,y:6),id:'6bf4b060-4201-11ea-8e3d-7f382b9d1c83',panelIndex:'52',type:visualization,version:'6.1.4-1'),(gridData:(h:3,i:'53',w:6,x:0,y:12),id:cf5abac0-765c-11ea-8fbd-d11a01cd3307,panelIndex:'53',type:visualization,version:'6.1.4-1')),query:(language:lucene,query:(query_string:(analyze_wildcard:!t,default_field:'*',query:'*'))),timeRestore:!t,title:Overview,uiState:(P-47:(spy:(mode:(fill:!f,name:!n)),vis:(params:(config:(searchKeyword:''),sort:(columnIndex:!n,direction:!n)))),P-48:(vis:(legendOpen:!t)),P-51:(vis:(legendOpen:!t)),P-53:(vis:(colors:('Aditya%20Sood':%23B7DBAB,'Emmanuel%20Engelhart':%230A50A1,'Geoffrey%20Kantaris':%23DEDAF7,'Isaac%20Hutt':%23BF1B00,'Joseph%20Reeve':%23F9934E,'Justin%20Biggs':%23705DA0,'Matthieu%20Gautier':%23508642,'Sean%20Mac%20Gillicuddy':%23967302,'Travis%20Briggs':%23D683CE,'renaud%20gaudin':%236ED0E0)))),viewMode:view)

But if i launch `cloc` on my local version of the repository, I get something around 5000 lines which seems more reasonable:
```
$ cloc .
     100 text files.
      98 unique files.                              
      10 files ignored.

github.com/AlDanial/cloc v 1.82  T=0.13 s (726.4 files/s, 256735.0 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
JSON                             9              0              0          24419
TypeScript                      44            898            273           5139
CSS                              4            104             53            435
Markdown                         3             67              0            223
JavaScript                       9             31             21            149
HTML                             9              6              6            120
YAML                             6             12             21             81
Bourne Shell                     5              3              5             44
Dockerfile                       1              4              3             22
PHP                              1              6              4             14
-------------------------------------------------------------------------------
SUM:                            91           1131            386          30646
-------------------------------------------------------------------------------
```

**Comment by florentk** [comment]
@kelson42 probably relation with https://github.com/kiwix/metrics/issues/60

**Comment by florentk** [comment]
fix with #60
